cmd/hare/oracle.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"github.com/spacemeshos/go-spacemesh/common/types"
+)
+
+type hareOracle struct {
+	oc *oracleClient
+}
+
+func newHareOracleFromClient(oc *oracleClient) *hareOracle {
+	return &hareOracle{oc: oc}
+}
+
+func (bo *hareOracle) IsIdentityActiveOnConsensusView(string, types.LayerID) (bool, error) {
```

Comment:
don't need to comment on that?

Reply:
no, `hareOracle` is not exported, so golint ignores it.